docs/version-overrides.md

```diff
@@ -44,3 +66,5 @@ This document lists the version overrides for vulnerable (nested) dependencies a
 :warning: Attention :warning: 
 * `trim`, `trim-newlines` and `glob-parent` are dependencies of `storybook` that is used in `@sap-ux/ui-components`. Once a new version without the vulnerable dependency is available, it is to be used and this override can be removed.
 * `semver` is used by too many modules. Override can be removed or kept for specific dependencies after we have cleaned up our devDependencies.
+* `tough-cookie` no fix available yet for jest-environment-jsdom
```

Comment:
No requirement to add an override only need to rebuild the lock file as the dependencies  to tough cookie are all open ranges.